atomate/vasp/fireworks/core.py

```diff
@@ -130,6 +134,42 @@ def __init__(self, structure, name="static", vasp_input_set=None,
             structure.composition.reduced_formula, name), **kwargs)
 
 
+class StaticInterpolateFW(Firework):
```

Comment:
can you refactor / move this to fireworks/ferroelectric.py (or something like that)?

Reply:
or fireworks/interpolate.py?